docs/content/contributing/submitting-issues.md

```diff
@@ -5,44 +5,55 @@ description: "Help us help you! Learn how to submit an issue, following the guid
 
 # Submitting Issues
 
-Help Us Help You!
+### Help Us Help You!
 {: .subtitle }
 
-We use the [GitHub issue tracker](https://github.com/traefik/traefik/issues) to keep track of issues in Traefik. 
+Issues are perfect for requesting a feature/enhancment or reporting a suspected bug. We use the [GitHub issue tracker](https://github.com/traefik/traefik/issues) to keep track of issues in Traefik. 
 
 The process of sorting and checking the issues is a daunting task, and requires a lot of work (more than an hour a day ... just for sorting).
-To save us some time and get quicker feedback, be sure to follow the guide lines below.
+To help us (and other community members) quickly and easily understand what you need, be sure to follow the guidelines below. 
 
 !!! important "Getting Help Vs Reporting an Issue"
 
     The issue tracker is not a general support forum, but a place to report bugs and asks for new features.
 
-    For end-user related support questions, try using first:
+    For end-user related support questions, try using the [Traefik Community Forum] (https://community.traefik.io/)
+ [![Join the chat at https://community.traefik.io/](https://img.shields.io/badge/style-register-green.svg?style=social&label=Discourse)](https://community.traefik.io/)
 
-    - the Traefik community forum: [![Join the chat at https://community.traefik.io/](https://img.shields.io/badge/style-register-green.svg?style=social&label=Discourse)](https://community.traefik.io/)
 
 ## Issue Title
 
 The title must be short and descriptive. (~60 characters)
 
-## Description
+Examples:
+          
+          * Bug: Duplicate requests in access logs
+          * FR:  Support fastCGI 
+            
+            
+## Feature Request
 
-Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.
+Traefik is an open source project and aims to be the best edge router possible. 
 
-Explain us in which conditions you encountered the issue, what is your context.
+Remember when asking for new features that these must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups). Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.
 
-Remain as clear and concise as possible
+Do your best to explain what you're looking for, and why it would improve Traefik for everyone. Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible. Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) naturally rise to the top of our to-do list.  
```

Comment:
```suggestion
Do your best to explain what you're looking for, and why it would improve Traefik for everyone.
Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible.
Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) naturally rise to the top of our to-do list.  
```